dev-python/mutagen/mutagen-1.47.0.recipe

```diff
@@ -27,9 +27,9 @@ BUILD_REQUIRES="
 	haiku_devel
 	"
 
-PYTHON_PACKAGES=(python39 python310)
-PYTHON_VERSIONS=(3.9 3.10)
-commandSuffixes=(3.8 "" 3.10)
+PYTHON_PACKAGES=(python39 python310 python311 python312)
+PYTHON_VERSIONS=(3.9 3.10 3.11 3.12)
+commandSuffixes=(3.9 "" 3.11 3.12)
```

Comment:
So far nothing has been pushed to python3.11 and 3.12, do we really need those?
Me pokes @OscarL